largest_sum_contiguous_subarray/LargestSumContiguousSubarray.cpp

```diff
@@ -0,0 +1,33 @@
+/*
+ *   Implementation of famous dynamic programming problem
+ *   Largest Sum Contiguous Subarray
+ *   Kadane Algorithm
+ *   Time complexity  O(n)
+ */
+#include <iostream>
+using namespace std;
+
+int largestSumContiguousSubarray(int length, int arr[]) {
+    int prevSum;
+    int currentSum;
+    int i;
+    prevSum = arr[0];						    // initialize current sum and previous sum

```

Comment:
This maybe just my opinion but I think you are overusing whitespace to indent the comment block here. 
Just a normal *consistent* one or two space inline comment works fine. 

```C
prevSum = arr[0];  // initialize current sum and previous sum
```

Though your style of commenting might feel good for the eye, it can actually cause a lot of problems when you are modifying the code. Not to mention, it also increases file-size to a certain degree and may cause horizontal scrollbars to appear even when they weren't needed. 
Views?